esp/src/eclwatch/LZBrowseWidget.js

```diff
@@ -393,8 +393,6 @@ define([
 
         _onSprayXml: function(event) {
             var context = this;
-            this.sprayXmlTargetRowTag = registry.byId(this.id + "sprayXmlTargetRowTag");
-            if (this.sprayXmlTargetRowTag.getValue()) {
```

Comment:
It was there because in the previous implementation due to thinking that per editor field you can assign an id hence https://github.com/hpcc-systems/HPCC-Platform/pull/8301/files#diff-be8a2914ef89d2e8a7ff85af53f63ff4L658 but it only does it for the first instance.